build-system/sauce_connect/start_sauce_connect.sh

```diff
@@ -31,6 +35,24 @@ READY_FILE="sauce_connect_ready"
 READY_DELAY_SECS=60
 LOG_PREFIX=$(YELLOW "start_sauce_connect.sh")
 
+# Check the status of the Sauce Labs service
+echo "$LOG_PREFIX Fetching current Sauce Labs service status from $(CYAN "$STATUS_URL")..."
+SAUCE_STATUS="$(curl -s "$AUTHENTICATED_STATUS_URL")"
+SERVICE_OPERATIONAL="$(echo "$SAUCE_STATUS" | jq '.service_operational')"
```

Comment:
`jq` is a missing dependency on my workstation. Add it to `devDependencies`?

Reply:
It's shipped with the default Travis VM

Reply:
It could still break the flow for people running `gulp pr-check` locally who have a sauce_labs key

Reply:
That workflow isn't supported, since this script is Linux-only.

We could change the local flow to install the platform appropriate binary, at which point `jq` can be added to the deps. (Out of scope here.)

Reply:
Fair enough